Penteract/Assets/Scripts/HUDController.cpp

```diff
@@ -315,23 +333,27 @@ void HUDController::UpdateCommonSkill() {
 	}
 }
 
-void HUDController::UpdateFangCooldowns(GameObject* fangSkillCanvas) {
-	std::vector<GameObject*> children = fangSkillCanvas->GetChildren();
+
+void HUDController::UpdateFangCooldowns(GameObject* fangSkillCanvas, bool isMain) {
+	std::vector<GameObject*> skills = fangSkillCanvas->GetChildren();
 	int skill = FANG_SKILL_1;
```

Comment:
!!!!!! Skill value never changes so it is always going to be calling AbilityCooldownEffectCheck(1..)

Reply:
It does change in line 365!